src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,118 @@
+            } else {
+                return Optional.empty();
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException("Can't create new manufacturer",e);
+        }
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        List<Manufacturer> result = new ArrayList<>();
+        String insertManufacturer = "SELECT * FROM manufacturer WHERE is_deleted = 0";
+        try (Connection connection = ConnectionUtil.getConnection();
+                Statement getAllManufacturerStatement
+                        = connection.createStatement()) {
+            ResultSet resultSet = getAllManufacturerStatement.executeQuery(insertManufacturer);
+            while (resultSet.next()) {
+                Long resultId = resultSet.getLong("id");
+                String name = resultSet.getString("name");
+                String country = resultSet.getString("country");
+                result.add(new Manufacturer(resultId, name, country));
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException("Can't create new manufacturer",e);
+        }
+        return result;
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        String insertManufacturer = "UPDATE manufacturer SET name = ?,country = ? WHERE id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement updateManufacturerStatement
+                         = connection.prepareStatement(insertManufacturer,
+                        Statement.RETURN_GENERATED_KEYS)) {
+            updateManufacturerStatement.setString(1,manufacturer.getName());
+            updateManufacturerStatement.setString(2,manufacturer.getCountry());
+            updateManufacturerStatement.setLong(3,manufacturer.getId());
+            updateManufacturerStatement.executeUpdate();
+            ResultSet generatedKeys = updateManufacturerStatement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                return manufacturer;
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException("Can't create new manufacturer",e);
+        }
+        return null;
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String insertManufacturer = "UPDATE manufacturer SET is_deleted = 1 WHERE id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement updateManufacturerStatement
+                        = connection.prepareStatement(insertManufacturer,
+                        Statement.RETURN_GENERATED_KEYS)) {
+            updateManufacturerStatement.setLong(1,id);
+            updateManufacturerStatement.executeUpdate();
+            ResultSet generatedKeys = updateManufacturerStatement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                return true;
+            }
```

Comment:
```suggestion
            return updateManufacturerStatement.executeUpdate() > 0;
```

I think it looks better this way